pallets/chainbridge/src/types.rs

```diff
@@ -3,7 +3,7 @@
 
 #![deny(warnings)]
 
-use codec::{
+use parity_scale_codec::u{
```

Comment:
Fatfingering with `u`? Or global rename? :)

Reply:
same: this file is not in module tree... should I remove it @Gauthamastro ?

Reply:
removed